subsys/shell/backends/shell_telnet.c

```diff
@@ -85,12 +85,31 @@ static void telnet_reply_ay_command(void)
 	telnet_command_send_reply((uint8_t *)alive, strlen(alive));
 }
 
+static int telnet_echo_set(const struct shell *sh, bool val)
+{
+	int ret = shell_echo_set(sh_telnet->shell_context, val);
+
+	if (ret < 0) {
+		LOG_ERR("Failed to disable echo, err: %d", ret);
```

Comment:
Please fix: This comment should probably be adjusted to something along the lines of `"Failed to set echo to: %d, err: %d"`?